Formula/node.rb

```diff
@@ -55,31 +57,22 @@ def install
     system "make", "install"
 
     if build.with? "npm"
-      resource("npm").stage buildpath/"npm_install"
-
-      # make sure npm can find node
+      # Allow npm to find Node before installation has completed.
       ENV.prepend_path "PATH", bin
-      # set log level temporarily for npm's `make install`
-      ENV["NPM_CONFIG_LOGLEVEL"] = "verbose"
-      # unset prefix temporarily for npm's `make install`
-      ENV.delete "NPM_CONFIG_PREFIX"
-
-      cd buildpath/"npm_install" do
-        system "./configure", "--prefix=#{libexec}/npm"
-        system "make", "install"
-        # `package.json` has relative paths to the npm_install directory.
-        # This copies back over the vanilla `package.json` that is expected.
-        # https://github.com/Homebrew/legacy-homebrew/issues/46131#issuecomment-157845008
-        cp buildpath/"npm_install/package.json", libexec/"npm/lib/node_modules/npm"
-        # Remove manpage symlinks from the buildpath, they are breaking bottle
-        # creation. The real manpages are living in libexec/npm/lib/node_modules/npm/man/
-        # https://github.com/Homebrew/legacy-homebrew/pull/47081#issuecomment-165280470
-        rm_rf libexec/"npm/share/"
-      end
+
+      system "/usr/bin/tar", "-xvf", resource("npm").cached_download
+      system "#{buildpath}/package/bin/npm-cli.js", "install",
+             "--verbose", "--global", "--prefix=#{libexec}",
+             resource("npm").cached_download
+      # We never use these symlinks & they've caused issues in the past.
+      rm_rf libexec/"share"
+      # package.json has relative paths to the package directory.
```

Comment:
I keep getting unicorned, sigh. GitHub status says things are fine though, so, eh.

Will push some tweaks here & sync it back up with the other two Nodes in this PR since we all seem reasonably happy this isn't an awful middle-ground between our positions.